arch/arm/src/at32/at32_adc.c

```diff
@@ -2458,6 +2458,7 @@ static int adc_setup(struct adc_dev_s *dev)
 
   if (priv->initialized > 0)
     {
+      priv->initialized += 1;
```

Comment:
```suggestion
      priv->initialized++;
```

Reply:
this file uses `priv->initialzied +=` in other places so I prefer to leave it as it is. The same for `stm32_adc.c`

Reply:
Ok